go_attack_utils/sample_training_games.py

```diff
@@ -0,0 +1,102 @@
+import re
+import glob
+import numpy as np
+import random
+import json
+
+random.seed(42)
+
+adv_pattern = re.compile("adv-s[0-9]+")
+victim_list = ["cp39h-v1", "cp127h-v1", "cp505h-v1", "cp505h-v2048"]
+
+
+def get_victim(input_text_line):
+    for victim in victim_list:  # go through possible victims
+        if victim in input_text_line:
+            return victim
+
+    # if none found, something is wrong
+    raise Exception("Victim not identified")
+
+
+all_files = glob.glob(
+    "/nas/ucb/k8/go-attack/match/ttseng-hard-adv-checkpoint-sweep-497mil-221115/ttseng-hard-match-*/sgfs/*.sgfs"

```

Comment:
That would be better, but will take a few hours for me to go through the new games and make sure the analysis stays the same (one because new random sample, two because the deciles will shift), update figure in paper, etc. Suggest we merge this for now so we can have updated site before FT article, and I'll make a new PR with this change (and corresponding paper updates) in next couple days.